website/docs/r/cloudwatch_event_target.html.markdown

```diff
@@ -68,11 +68,11 @@ The following arguments are supported:
 
 * `rule` - (Required) The name of the rule you want to add targets to.
 * `target_id` - (Optional) The unique target assignment ID.  If missing, will generate a random, unique id.
-* `arn` - (Required) The Amazon Resource Name (ARN) associated of the target.
+* `arn` - (Required) The Amazon Resource Name (ARN) associated of the target. (It should be the ID of the ECS Cluster if `ecs_target` is used.)
 * `input` - (Optional) Valid JSON text passed to the target.
 * `input_path` - (Optional) The value of the [JSONPath](http://goessner.net/articles/JsonPath/)
 	that is used for extracting part of the matched event when passing it to the target.
-* `role_arn` - (Optional) The Amazon Resource Name (ARN) of the IAM role to be used for this target when the rule is triggered.
+* `role_arn` - (Optional) The Amazon Resource Name (ARN) of the IAM role to be used for this target when the rule is triggered. (Required if `ecs_target` is used.)
```

Comment:
Sorry, loved aussi those brackets, was meant to remove parenthesis, or `(`. Dunno bracket was the same for ` & `(` or `)`
So would go for `Required if `ecs_target` is used.`, simply :D

Sorry for the extra work ahah!